File: xyident/baekjoon/9단계/1978.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

num = input[1].split(' ');
// num = [1, 3, 5, 7];
let count = 0;

for(let i = 0; i < num.length; i++){
    let factors = 0;
    
    for(let j = 1; j < num[i]; j++){
        if(num[i] % j == 0){
            factors++;
        }
    }

    if(num[i] > 1 && factors == 1){
        count++;
    } 
}

console.log(count);

File: rolemadelen/baekjoon/step9/1978.rs
use std::io::{stdin};

fn read() -> i32 {
    let mut input = String::new();
    stdin().read_line(&mut input).unwrap();
    input.trim().parse().unwrap()
}

fn is_prime(x: i32) -> bool {
    if x < 2 {
        return false;
    }
    if x == 2 {
        return true;
    }
    if x % 2 == 0 {
        return false;
    }

    for i in (3..x).step_by(2) {
        if x % i == 0 {
            return false;
        }
    }

    true
}

fn main() {
    read();
    let mut input = String::new();
    stdin().read_line(&mut input).unwrap();
    let nums: Vec<i32> = input.trim().split(' ').map(|x| x.trim().parse().unwrap()).collect();

    let mut cnt = 0;
    for x in nums {
        if is_prime(x) {
            cnt += 1;
        }
    }

    println!("{}", cnt);
}
